module1/build tools/multi-project/src/main/java/Main.java

```diff
@@ -0,0 +1,6 @@
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
Why do you need main method is this class?

Reply:
Reduild project, deleted class.